Examples/SwiftExample/OneSignalDemo/AppDelegate.swift

```diff
@@ -43,30 +43,34 @@ class AppDelegate: UIResponder, UIApplicationDelegate, OSPermissionObserver, OSS
         // For debugging
         OneSignal.setLogLevel(.LL_VERBOSE, visualLevel: .LL_NONE)
         
-        let notificationReceivedBlock: OSHandleNotificationReceivedBlock = { notification in
-            
-            print("Received Notification: ", notification!.payload.notificationID!)
-            print("launchURL: ", notification?.payload.launchURL ?? "No Launch Url")
-            print("content_available = \(notification?.payload.contentAvailable ?? false)")
+        let notifWillShowInForegroundHandler: OSNotificationWillShowInForegroundBlock = { notification, completion in
+            print("Received Notification: ", notification.notificationId ?? "no id")
+            print("launchURL: ", notification.launchURL ?? "no launch url")
+            print("content_available = \(notification.contentAvailable)")
+            if notification.notificationId == "example_silent_notif" {
+                completion(OSNotificationDisplayType.silent)
+            } else {
+                completion(OSNotificationDisplayType.notification)
```

Comment:
Will need to update after we remove `OSNotificationDisplayType` but we can merge this PR and come back with a follow up PR.